### DISCUSSION TITLE
How to use Mapsui in the Windows Form framework

### DISCUSSION THREAD

**Discussion by lucinelouhi** [Mapsui/Mapsui#2614] [Q&A]
How to use Mapsui in the Windows Form framework. Or consider adding Windows Form support. Because some computers may not have high performance, using WinUI or Wpf may cause lag. The versatility of Windows Forms is better.

**Comment by pauldendulk** [comment]
See my answer here: https://github.com/Mapsui/Mapsui/discussions/2613.

About performance. Most of the performance hit will be in rendering the map and fetching the data, so in that respect it might not be much better.
